.github/workflows/test.yml

```
@@ -0,0 +1,44 @@
+name: lint and test
+
+on:
+   push:
+      branches:
+         - master
+   pull_request:
+      branches:
+         - master
+
+jobs:
+   eslint:

```

eslintという名前のフロー内部で全部の処理がされている。
```suggestion
   なんか良さげな名前:

```